frontend/ContextMenu.js

```diff
@@ -11,12 +11,12 @@
 'use strict';
 
 var React = require('react');
-var {sansSerif} = require('./Themes/Fonts');
+var { sansSerif } = require('./Themes/Fonts');
 var HighlightHover = require('./HighlightHover');
 
 var decorate = require('./decorate');
 
-import type {Theme} from './types';
+import type {Theme } from './types';
```

Comment:
The spacing should be fixed, I believe that was due to vscode's selection formatting.